roles/installer/tasks/resources_configuration.yml

```diff
@@ -245,17 +245,59 @@
   set_fact:
     _redis_image: "{{ _custom_redis_image | default(lookup('env', 'RELATED_IMAGE_AWX_REDIS')) | default(_default_redis_image, true) }}"
 
-- name: Apply deployment resources
+- name: Create directory for kustomize
+  ansible.builtin.file:
+    path: "deployments"
+    state: directory
+  register: _deployment_kustomize_dir
+
+- name: Create tmp dir for {{ item }} deployment
+  tempfile:
+    state: directory
+  register: _deployment_kustomize_dir
+
+- name: Copy over kustomization.yaml to tmp dir
+  copy:
+    src: ../templates/deployments/kustomization.yaml
+    dest: "{{ _deployment_kustomize_dir.path }}"
+
+- name: Template deployments and overrides
+  template:
+    src: deployments/{{ item }}.yaml.j2
+    dest: "{{ _deployment_kustomize_dir.path }}/{{ item }}.yaml"
+  loop:
+    - "task"
+    - "task_override"
+    - "web"
+    - "web_override"
+
+- name: Run kustomize build on task deployment and store in deployments.yaml
+  command: kustomize build {{ _deployment_kustomize_dir.path }} -o {{ _deployment_kustomize_dir.path }}/deployments.yaml
```

Comment:
We can use `kubernetes.core.kustomize` lookup instead.

https://docs.ansible.com/ansible/latest/collections/kubernetes/core/kustomize_lookup.html